indico/core/db/sqlalchemy/locations.py

```diff
@@ -248,3 +248,48 @@ def location_data(self):
             return {'source': data_source, 'venue': data_source.venue, 'room': data_source.room,
                     'room_name': data_source.room_name, 'venue_name': data_source.venue_name,
                     'address': data_source.address, 'inheriting': self.inherit_location}
+
+    @property
+    def location_data_for_ui(self):
```

Comment:
If you keep it here I'd at least rename it to `widget_location_data`. "for" in a variable/function name usually means it's too verbose